### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by bertho-zero**
This PR aims to deduplicate the Diva download step, it fixes the "`cd diva-alpha-net` inside the `diva-alpha-net` folder" and it fixes the "previous" and "next" links which were broken.

### CODE REVIEW

**Comment by micho** [comment]
Nice one :) I support this PR, as the previous one was duplicating the `cd` commands

File: docs/testnet/install/download.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+hide_table_of_contents: false
+---
+
+# Download Diva package
+
+Type the following commands in the CLI of a UNIX based system to clone this repository to your local machine:
+
+```bash
+# Clone the repository
+git clone https://github.com/shamirlabs/diva-alpha-net
```
Comment (pradavc) [inline docs/testnet/install/download.md:12]:
```suggestion
git clone https://github.com/shamirlabs/diva-alpha-net ~/diva-alpha-net
```

File: docs/testnet/install/download.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Change directory
+cd diva-alpha-net
```
Comment (pradavc) [inline docs/testnet/install/download.md:15]:
```suggestion
```

File: docs/testnet/install/scripts/install-diva.md [hunk 3]
```diff
@@ -8,7 +8,6 @@ hide_table_of_contents: false
 Run the installation script by executing:
 
 ```bash
-cd diva-alpha-net
 ./run.sh
```
Comment (pradavc) [inline docs/testnet/install/scripts/install-diva.md:11]:
```suggestion
cd ~/diva-alpha-net
./run.sh
```

File: docs/testnet/install/scripts/install-docker.md [hunk 4]
```diff
@@ -8,7 +8,6 @@ hide_table_of_contents: false
 Run the installation script by executing:
 
 ```bash
-cd diva-alpha-net
 ./run.sh
```
Comment (pradavc) [inline docs/testnet/install/scripts/install-docker.md:11]:
```suggestion
cd ~/diva-alpha-net
./run.sh
```

File: docs/testnet/update-diva.md [hunk 5]
```diff
@@ -8,7 +8,6 @@ hide_table_of_contents: false
 Run the installation script by executing:
 
 ```bash
-cd diva-alpha-net
 ./run.sh
```
Comment (pradavc) [inline docs/testnet/update-diva.md:11]:
```suggestion
cd ~/diva-alpha-net
./run.sh
```

**Review by pradavc — CHANGES_REQUESTED** [review]
The `cd diva-alpha-net` lines were not duplicated but placed intentionally to make all the commands work agnostic to the path from which the user executes them. Relative paths were used in the previous version because I didn't want to enforce a strict path for the clone in users' systems.

Since you want to avoid confusion (and I agree that for this guide level, it is better), we can assume that the clone is done to the home path. In that case, it is better to remove the `cd` line after the `git clone`, which is useless, enforce cloning to the home path, and keep the `cd` command lines inside the doc of each option of the script (update, install, etc). That way, no matter where the user starts reading, it will always work (even if it comes back directly for the update instructions only, which currently assumes that the user is under `$HOME/diva-alpha-net` when it isn't).

**Comment by micho** [comment]
lgtm

**Pull request merged**